daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/LedgerInteraction.scala

```diff
@@ -427,10 +428,10 @@ class IdeClient(val compiledPackages: CompiledPackages) extends ScriptLedgerClie
                     // Capture the result and exit.
                     result = results.toSeq
                 }
-              case Right(Left(x)) =>
-                throw new RuntimeException(s"Unexpected abort: $x")
-              case Left(msg) =>
-                throw new RuntimeException(msg)
+              case PartialTransaction.IncompleteTransaction(ptx) =>
+                result = Failure(new RuntimeException(s"Unexpected abort: $ptx"))
```

Comment:
How does this work? `result` is no longer a `Try[…]`, as of #7179.

Reply:
fixed.